python/mxnet/module/executor_group.py

```diff
@@ -273,9 +273,10 @@ def __init__(self, symbol, contexts, workload, data_shapes, label_shapes, param_
         self.data_layouts = None
         self.label_layouts = None
         self.output_names = self.symbol.list_outputs()
-        self.output_layouts = [DataDesc.get_batch_axis(self.symbol[name].attr('__layout__'))
-                               for name in self.output_names]
-        self.num_outputs = len(self.symbol.list_outputs())
+        self.num_outputs = len(self.output_names)
+        self.output_layouts = [i for i in range(self.num_outputs)]
```

Comment:
Thank you for the contribution!
The following code may be faster : )
```python
        self.num_outputs = len(self.output_names)
        self.output_layouts = [DataDesc.get_batch_axis(self.symbol[index].attr('__layout__'))
                               for index in range(self.num_outputs)]
```

Reply:
Thanks!